lib/environments/meteorEnvironment.js

```diff
@@ -276,24 +276,16 @@ function extractExportsFromMeteorPackage(
     }
 
     buildIsopack.declaredExports.forEach((declaredExport: Object) => {
-      if (declaredExport.testOnly) {
-        return;
+      if (!declaredExport.testOnly) {
+        declaredExports.add(declaredExport.name);
       }
-      if (declaredExports.indexOf(declaredExport.name) !== -1) {
-        return;
-      }
-
-      // This declaredExport is not a "testOnly" export and is not already
-      // amongst those we've gathered, so we want to push it into our
-      // collection.
-      declaredExports.push(declaredExport.name);
     });
 
     // TODO: If the "resources" section of the buildIsopack specifies a
     // mainModule, we need to attempt to scan it to find exports.
   });
 
-  return declaredExports;
+  return Array.from(declaredExports);
```

Comment:
Yep, though I've been following a PR that fixes it - facebook/flow#1668.

That syntax seems to be what the community is converging on as most common for the conversion. It's also used in most examples of how to work around the lack of a built-in union operator,,, like [this one](http://www.2ality.com/2015/01/es6-set-operations.html).

As Pascal was my first "real" language (circa 1982 after Cobol, Basic, and some assembly), I learned to appreciate sets early on, especially in language processing applications.  You'll probably see me using more of them.